languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35467,6 +35467,19 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rule id="QU_EST_CE_QUE" name="qu est-ce que" default="temp_off">
+          <pattern>
+              <marker>
+                  <token case_sensitive="yes">qu'</token>
+                  <token>est</token>
+                  <token>-ce</token>
+                  <token regexp="yes">qu.*</token>
+              </marker>
+          </pattern>
```

Comment:
This pattern now matches any occurrence of _qu'est-ce que/qu'_.

This will cause FPs for:

- Sinon, qu'est-ce qu'on va manger aujourd'hui ?
- Paul, qu'est-ce que tu as encore fait ?

This rule is now too general. You'll have to avoid going just for any "qu'est-ce que" and define some kind of context.